lib/common.js

```diff
@@ -99,5 +123,6 @@ module.exports = { unescapeString: unescapeString,
                    OPENTAG: OPENTAG,
                    CLOSETAG: CLOSETAG,
                    ENTITY: ENTITY,
-                   ESCAPABLE: ESCAPABLE
+                   ESCAPABLE: ESCAPABLE,
+                   objectAssign: Object.assign
```

Comment:
Can you explain the `Object.assign` business?  (I'm not a JS expert.)
Is this a shim to cover for the absence of a new JS feature on older browsers?